scripts/release/validate-appimage-runtime.sh

```
@@ -30,23 +30,149 @@ runtime_validation_error() {
+  echo "[appimage-runtime] desktop entry ($desktop):"
+  sed 's/^/[appimage-runtime]   /' "$desktop"
+  local key
+  for key in Exec Icon Name; do
+    grep -q "^${key}=" "$desktop" \
+      || { runtime_validation_error "desktop entry is missing ${key}="; return 1; }
+  done
+
+  # -e follows symlinks: the AppDir-root icon and .DirIcon are relative symlinks
+  # into usr/share/icons, NOT regular files.
+  local link
+  for link in "$appdir/.DirIcon"; do
```

Shellcheck (0.11.0)

[error] Since you double quoted this, it will not word split, and the loop will only run once.

(SC2066)